radixdlt-java/src/main/java/com/radixdlt/client/atommodel/rri/RRIParticle.java

```diff
@@ -24,14 +24,14 @@
 	private RRIParticle() {
 	}
 
-	public RRIParticle(RadixResourceIdentifer rri) {
+	public RRIParticle(RRI rri) {
 		super();
 
 		this.rri = Objects.requireNonNull(rri);
 		this.nonce = 0;
 	}
 
-	public RadixResourceIdentifer getRri() {
+	public RRI getRri() {
```

Comment:
Should be `getRRI()`.  Will also match with same name in `TokenDefinitionParticle`.